articles/openshift/howto-gpu-workloads.md

```diff
@@ -188,6 +188,27 @@ ARO uses Kubernetes MachineSet to create machine sets. The procedure below expla
 
 1. Verify the other data in the yaml file.
 
+#### Ensure the correct SKU is set
+
+Depending on the image used for the machine set, the value for `image.sku` must be set accordingly. This is to ensure if generation 1 or 2 virtual machine for Hyper-V will be used. More details [here](https://learn.microsoft.com/en-us/windows-server/virtualization/hyper-v/plan/should-i-create-a-generation-1-or-2-virtual-machine-in-hyper-v).
+
+Example:
+
+If using `Standard_NC4as_T4_v3` it is supported both versions, as mentioned at [Feature support](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/gpu-accelerated/ncast4v3-series?tabs=sizebasic#feature-support). So no changes is required.
+
+If using `Standard_NC24ads_A100_v4`, only **Generation 2 VM** is [supported](https://learn.microsoft.com/en-us/azure/virtual-machines/sizes/gpu-accelerated/nca100v4-series?tabs=sizebasic).
+In this case, the `image.sku` value must follow the equivalent `v2` version of the image that corresponds to the cluster's original `image.sku`. For this example, the value will be `v410-v2`.
+
+This can be found using below command:
+
+```bash
+az vm image list --all --offer aro4 --publisher azureopenshift
+```
```

Comment:
Two suggestions:
* Add some output of `az vm image list --architecture x64 -o table --all --offer aro4 --publisher azureopenshift` and highlight the required column
* use this command instead `az vm image list --architecture x64 -o table --all --offer aro4 --publisher azureopenshift` because tables are easier to read for humans an the `--architecture` selector reduce the runtime of the command :smiley:

Reply:
Makes sense, I adjusted the command and added a filtered output for example.